server/services/database/message.js

```diff
@@ -5,22 +5,29 @@ const pool = new Pool(config);
 const createMessage = ({ recruiter_id, profile_public_id, message }) => {
   return pool.query(
     "INSERT INTO message (recruiter_id, profile_public_id, message) values ($1, $2, $3)",
-    [recruiter_id, profile_public_id, message],
+    [recruiter_id, profile_public_id, message]
   );
 };
 
+const getMessagesByRecruiterId = (id) => {
+  return pool
+    .query("SELECT profile_public_id FROM message WHERE recruiter_id=$1", [id])
```

Comment:
This will return only `profile_public_id` and your function name is getMessagesByRecruiterId, I think it should be changed to a related name.

Reply:
Done. Thanks.